### PR TITLE
use CSS variables for extensionEditor

### PR DESCRIPTION
**Pull request by Kalmaegi**
for https://github.com/microsoft/vscode/issues/165169

### CODE REVIEW

File: src/vs/workbench/contrib/extensions/browser/extensionEditor.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ import { DEFAULT_MARKDOWN_STYLES, renderMarkdownDocument } from 'vs/workbench/co
 import { ILanguageService } from 'vs/editor/common/languages/language';
 import { TokenizationRegistry } from 'vs/editor/common/languages';
 import { generateTokensCSSForColorMap } from 'vs/editor/common/languages/supports/tokenization';
```
Comment (mlincyy) [inline src/vs/workbench/contrib/extensions/browser/extensionEditor.ts:60]: 222

Comment (mlincyy) [reply src/vs/workbench/contrib/extensions/browser/extensionEditor.ts:60]: 222

Comment (mlincyy) [reply src/vs/workbench/contrib/extensions/browser/extensionEditor.ts:60]: 222222

Comment (mlincyy) [reply src/vs/workbench/contrib/extensions/browser/extensionEditor.ts:60]: 222

Comment (mlincyy) [inline src/vs/workbench/contrib/extensions/browser/extensionEditor.ts:60]: 22222

Comment (mlincyy) [reply src/vs/workbench/contrib/extensions/browser/extensionEditor.ts:60]: 3333

Comment (mlincyy) [reply src/vs/workbench/contrib/extensions/browser/extensionEditor.ts:60]: 1111

Comment (mlincyy) [reply src/vs/workbench/contrib/extensions/browser/extensionEditor.ts:60]: 111

**Comment by sandy081** [comment]
Sorry that I am not accepting PRs for these changes as they are sensitive.

**Pull request closed without merging**